Explorer/Assets/DCL/AvatarRendering/AvatarShape/Components/AvatarTransformMatrixComponent.cs

```diff
@@ -16,6 +16,9 @@ public struct AvatarTransformMatrixComponent : IDisposable
 
         public void ScheduleBoneMatrixCalculation(Matrix4x4 avatarWorldToLocalMatrix)
         {
+            if (!Handle.IsCompleted)
```

Comment:
I'm out of the loop, but could this mitigation fix hide other errors? The code itself looks good to approve, but I was wondering about that.

Reply:
Hi Ale! Its a good point. I dont think its going to hide other errors, but rather complete the job when it has been stopped by a previous error. So, the error should still be there, but this one should not occur as the job will be completed on the next frame

Reply:
That makes sense, thanks for taking the time to add more context, I appreciate it!